test/controllers/api/v1/code_schools_controller_test.rb

```diff
@@ -29,6 +29,22 @@ class Api::V1::CodeSchoolsControllerTest < ActionDispatch::IntegrationTest
     assert errors.include? "Url can't be blank"
   end
 
+  test ':validates CodeSchool uses HTTPS for URL' do
+    params = {
+      code_school: {
+        name: '1337School',
+        url: 'http://31337codeschool.com'
+      }
+    }
+    post api_v1_code_schools_url,
+      params: params,
+      headers: @headers,
+      as: :json
+
+    errors = JSON.parse(response.body)['errors']
+    assert errors.include? 'Url must be HTTPS, if unable please secure your site'
```

Comment:
👏

Reply:
If you have to have HTTPS for a .dev domain (https://get.dev/#benefits), any coding school worth their salt should have it by default. Would be interested to hear use cases where it's HTTP, but I can't think of a reason to not have it nowadays.